.gitignore

```diff
@@ -62,12 +62,14 @@ install_manifest.txt
 # vim temp files
 *.swp
 *.un~
+*.*~
 
 # Emacs files:
 # VCS comparison files:
 *.~*~
 # Autosave files
 .#*
+#*#
```

Comment:
I get the other two changes, but isn't this just a comment as it begins with `#`?  Or am I not reading something correctly.